examples/07_join_aggregation.py

```diff
@@ -0,0 +1,438 @@
+"""
+TODO : Add some docstring
+----
+"""
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Movies recommendation with explicit labels
+
+# ## Loading data
+
+# In[1]:
+
+
+from skrub.datasets import fetch_movielens
+
+ratings = fetch_movielens(dataset_id="ratings")
+ratings = ratings.X
+ratings
+
+
+# In[2]:
+
+
+import pandas as pd
+
+ratings["timestamp"] = pd.to_datetime(ratings["timestamp"], unit="s")
+ratings["year"] = ratings["timestamp"].dt.year
+ratings["day_name"] = ratings["timestamp"].dt.day_name()
```

Comment:
I think the DatetimeEncoder does this once you give the `timestamp` column to the TableVectorizer